<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddChannelDialog</class>
 <widget class="QDialog" name="AddChannelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/picture/picture/003.png</normaloff>:/picture/picture/003.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_AddChannel">
     <property name="title">
      <string>新增通道参数配置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tableView_Channel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>366</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView_RecordChannel"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn__AddChannel">
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
